packages/app-mobile/components/ExtendedWebView/index.jest.tsx

```diff
@@ -46,6 +46,25 @@ const ExtendedWebView = (props: Props, ref: Ref<WebViewControl>) => {
 	injectedJavaScriptRef.current = props.injectedJavaScript;
 
 	useEffect(() => {
+		// JSDOM polyfills
+		dom.window.eval(`
+			// Prevents the CodeMirror error "getClientRects is undefined".
+			// See https://github.com/jsdom/jsdom/issues/3002#issue-652790925
+			document.createRange = () => {
+				const range = new Range();
+				range.getBoundingClientRect = () => {};
+				range.getClientRects = () => {
+					return {
+						length: 0,
+						item: () => null,
+						[Symbol.iterator]: () => {},
+					};
+				};
+
+				return range;
+			};
```

Comment:
This logic is currently also present in `jest.base-setup.js`.

https://github.com/laurent22/joplin/blob/2df8137281a65aa90ad4e9894e21e47fb9ddf85f/jest.base-setup.js#L22

To avoid duplicate logic, we could:
- `require()` a file with JSDOM polyfills in `jest.base-setup.js`
- `fs.readFile()` the contents of that file in WebView tests.

I think this polyfill is only required for packages that depend on the `codemirror` packages. Perhaps this polyfill could be included in `packages/editor/`?